orchestration/dagster_orchestration/hca_orchestration/pipelines/load_hca.py

```diff
@@ -78,6 +79,7 @@
 )
 def load_hca() -> None:
     staging_dataset = create_scratch_dataset(pre_process_metadata(clear_scratch_dir()))
-    # import_data_files(staging_dataset)
-    file_metadata_fanout(staging_dataset)
-    non_file_metadata_fanout(staging_dataset)
+    result = import_data_files(staging_dataset).collect()
```

Comment:
I am consuming a result here to induce a dependency between `import_data_files` and `file_metadata_fanout`, otherwise we load the metadata before file_ids are set.

Reply:
I looked into switching this to a ["nothing" dependency chain ](https://docs.dagster.io/concepts/solids-pipelines/pipelines#order-based-dependencies-nothing-dependencies)but I have bumped into what appears to be a dagster bug with composite solids. I'm chatting in their slack, but this works ok for now.